src/Symfony/Component/Security/Core/Tests/Validator/Constraints/UserPasswordValidatorTest.php

```diff
@@ -79,10 +83,11 @@ public function testPasswordIsNotValid()
             'message' => 'myMessage',
         ));
 
-        $this->encoder->expects($this->once())
+        $this->encoder
+            ->expects($this->once())
             ->method('isPasswordValid')
             ->with(static::PASSWORD, 'secret', static::SALT)
-            ->will($this->returnValue(false));
+            ->willReturn(false);
```

Comment:
switching to the new PHPUnit APIs should not be done in PRs cleaning deprecated layers. It is totally unrelated so it should be its own PR (and if we decide to do it, we should apply it in the 2.3 branch and then merge it to newer branches, otherwise it will become a huge pain for maintainers to merge branches together for the next 3 years)